### DISCUSSION TITLE
Select entity using multiple boolean dpid's

### DISCUSSION THREAD

**Discussion by blakadder** [make-all/tuya-local#896] [Q&A] [answered]
Is it possible and if it is, how to create a select entity that uses multiple dpids (boolean type generally) as select options?

**Comment by make-all** [comment, marked as answer]
You can go as far as combining two dps using conditional mappings.  Beyond that the configuration processing gets exponentially more complex with each addtional dp added, and there hasn't been any use case identified so more complex combinations have not been implemented.
